build.info

```diff
@@ -11,11 +11,95 @@ DEPEND[libssl]=libcrypto
+DEPEND[doc/man1/openssl-cms.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-cms.pod]=doc/man1/openssl-cms.pod.in
+DEPEND[doc/man1/openssl-crl.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-crl.pod]=doc/man1/openssl-crl.pod.in
+DEPEND[doc/man1/openssl-dgst.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-dgst.pod]=doc/man1/openssl-dgst.pod.in
+DEPEND[doc/man1/openssl-dhparam.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-dhparam.pod]=doc/man1/openssl-dhparam.pod.in
+DEPEND[doc/man1/openssl-dsaparam.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-dsaparam.pod]=doc/man1/openssl-dsaparam.pod.in
+DEPEND[doc/man1/openssl-ecparam.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-ecparam.pod]=doc/man1/openssl-ecparam.pod.in
+DEPEND[doc/man1/openssl-enc.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-enc.pod]=doc/man1/openssl-enc.pod.in
+DEPEND[doc/man1/openssl-gendsa.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-gendsa.pod]=doc/man1/openssl-gendsa.pod.in
+DEPEND[doc/man1/openssl-genrsa.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-genrsa.pod]=doc/man1/openssl-genrsa.pod.in
+DEPEND[doc/man1/openssl-ocsp.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-ocsp.pod]=doc/man1/openssl-ocsp.pod.in
+DEPEND[doc/man1/openssl-passwd.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-passwd.pod]=doc/man1/openssl-passwd.pod.in
+DEPEND[doc/man1/openssl-pkcs12.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-pkcs12.pod]=doc/man1/openssl-pkcs12.pod.in
+DEPEND[doc/man1/openssl-pkcs8.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-pkcs8.pod]=doc/man1/openssl-pkcs8.pod.in
+DEPEND[doc/man1/openssl-pkeyutl.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-pkeyutl.pod]=doc/man1/openssl-pkeyutl.pod.in
+DEPEND[doc/man1/openssl-rand.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-rand.pod]=doc/man1/openssl-rand.pod.in
+DEPEND[doc/man1/openssl-req.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-req.pod]=doc/man1/openssl-req.pod.in
+DEPEND[doc/man1/openssl-rsautl.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-rsautl.pod]=doc/man1/openssl-rsautl.pod.in
+DEPEND[doc/man1/openssl-s_time.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-s_time.pod]=doc/man1/openssl-s_time.pod.in
+DEPEND[doc/man1/openssl-smime.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-smime.pod]=doc/man1/openssl-smime.pod.in
+DEPEND[doc/man1/openssl-speed.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-speed.pod]=doc/man1/openssl-speed.pod.in
+DEPEND[doc/man1/openssl-srp.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-srp.pod]=doc/man1/openssl-srp.pod.in
+DEPEND[doc/man1/openssl-ts.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-ts.pod]=doc/man1/openssl-ts.pod.in
+DEPEND[doc/man1/openssl-verify.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-verify.pod]=doc/man1/openssl-verify.pod.in
+DEPEND[doc/man1/openssl-x509.pod.in]=doc/perlvars.pm
+GENERATE[doc/man1/openssl-x509.pod]=doc/man1/openssl-x509.pod.in
```

Comment:
This is such an amount that I think it would be worth moving to `doc/man1/build.info`

Reply:
It might also be worth revamping my doc-build-at-build-time PR

Reply:
Happy to do whatever you want, just let me know.  I'll just leave this here:
```
perl -Mconfigdata -Mdoc/perlvars.pm -i.in doc/man1/*.in
```
:)

Reply:
That won't work

Reply:
Like i said, just tell me what you want me to do, once you figure it out.

Reply:
```
perl -Mconfigdata -Idoc -Mperlvars util/dofile.pl -i.in doc/man1/*.in
```

Reply:
But still, move those lines to doc/man1/build.info, and add this to the top build.info:

```
SUBDIRS=doc/man1
```

Reply:
done, additional commit pushed.